contents/docs/product-analytics/group-analytics.mdx

```diff
@@ -24,10 +24,110 @@ import JSSetGroupProperties from "../\_snippets/setting-group-properties.mdx"
 
 <JSCreateGroups />
 
-### Setting and updating group properties
+## How to set group properties
 
 <JSSetGroupProperties />
 
+## How to capture group events
+
+Events are automatically linked to the groups that the user who submitted the event belongs to. Thus you do not add any additional code once you have linked a user to the group. This is the recommended way to capturing group events.
+
+To link a user to a group, call the same code as you would when [creating a group](#how-to-create-groups).
+
+### Advanced (server-side only): Capturing group events without a user
+
+If you want to capture group events but don't want to associate them with a specific user, there are two alternatives:
+
+#### Method one (recommended): Use the group ID as the event's distinct ID
+
+When capturing events, instead of using the user's ID as the distinct ID, you can use group ID as the distinct ID. For example:
+
+<MultiLanguage>
+
+```node
+posthog.capture({
+    event: 'group_event_name',
+    distinctId: 'group_id_in_your_db',
+})
+```
+
+```python
+posthog.capture('group_id_in_your_db', 
+    'group_event_name', 
+)
+```
+
+```go
+client.Enqueue(posthog.Capture{
+    DistinctId: "group_id_in_your_db",
+    Event: "group_event_name",
+})
+```
+
+```php
+PostHog::capture(array(
+    'distinctId' => 'group_id_in_your_db',
+    'event' => 'group_event_name'
+));
+```
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "event": "group_event_name",
+    "distinct_id": "group_id_in_your_db",
+}' <ph_instance_address>/capture/
+```
+
+This has the advantage that insights such as "total users and "unique users" in [trends](/docs/product-analytics/trends) will work as expected for these groups events (note: although it will say "users", it will actually mean "groups")
+
+The drawback of this approach is that a new [person](/docs/data/persons) will be created for each group. While this won't affect your data integrity in any way, it may cause confusion since these groups are not actually people.
+
+</MultiLanguage>
+
+#### Method two: Use the same ID as the event's distinct ID
+
+Instead of using the group's ID as the distinct ID, you can create a *single string* that will be used to capture group events. This can be anything you want, as long as it's the same string for every group event:
```

Comment:
```suggestion
Instead of using the group's ID as the distinct ID, you can use a *single value* to capture group events. This can be anything you want, as long as it's the same for every group event:
```